model.py

```diff
@@ -3012,7 +3012,7 @@ class Edition(Base):
 
     # These are the media types currently fulfillable by the default
     # client.
-    FULFILLABLE_MEDIA = [BOOK_MEDIUM]
+    FULFILLABLE_MEDIA = [BOOK_MEDIUM, AUDIO_MEDIUM]
```

Comment:
This is a deceptively large change -- this means audio content will start showing up. It won't show up in the existing lane structure, because the existing lanes are hard-coded to only contain ebooks. But it will show up on the front page if the library administrator turns on that entry point (in which case it will have its own entry point) or turns off every single entry point (in which case it will be mixed with ebook content).

I'm OK with leaving this out for now but I'm using it to test on a real system.

The larger problem this causes is that Overdrive audio content starts showing up in feeds even though it's not currently fulfillable.

Reply:
I'm not sure what the implications of this are. I'm okay with having it in if the default is for only the ebook entry point to be enabled, and an admin would have to explicitly change something for audiobooks to show up.